content/millennium/dstu2/entities/contract.md

```diff
@@ -9,7 +9,19 @@ title: Contract | DSTU 2 API
 
 ## Overview
 
-The Contract resource usage is limited to representing persons that are authorized to view patient data. The subject of the contract is the patient that is the target of this authorization. The actors on the contract are the related people that are authorized to view data. An example would be where a contract represents that a mother is authorized to view her children's data. The contract is that authorization in FHIR.
+The Contract resource is currently limited to representing people who are authorized to view a patient's data. For this reason, contract type, subtype, action and reason are always set to values related to consent for EHR sharing. The subject of the contract is the patient. The actors on the contract are the related people who are authorized to view the subject patient's data. An example is a contract authorizing a mother (actor) to view her child’s data (subject). 
```

Comment:
The last apostrophe in this paragraph is a curved quote, but it seems to render the same on the site as the straight quotes